cdatransform/transform/transform_lib/transform_with_YAML_v1.py

```diff
@@ -3,102 +3,112 @@
 from cdatransform.transform.validate import LogValidation
 
 
```

Comment:
I'm just gonna ask that maybe when I get back from my vacation that we can do a walkthrough of the transformation code.   I know that we went over this a bit when I first came on the project.  It is a little hard to follow, so I think that doing a screencast of following through this code now that I know the data domain just a bit better.

Reply:
I agree that it is a bit hard to follow. I added a portion of code that would read from yaml files and some of the repo needs to be cleaned up. There are probably other better ways to organize libraries as well which would make it less confusing. I was going to setup a time with Elijah to go over the transformation code as well so hopefully we can setup a time with all of us

Reply:
That sounds great to me.  I added Dion as a reviewer and he's going to go over that right now and will sign off on it for you since I'll be gone next week.